src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -498,7 +498,1246 @@ The user can access any of the four portals using the default credentials: `admi
 
 Your ProActive Scheduler is now running as a set of container services!
 
+==== How to install ProActive using Kubernetes
 
+ProActive could be easily deployed and started in a Kubernetes cluster. For this purpose, we provide a ready-to-use Kubernetes configuration files that enable you running ProActive Workflows and Scheduling as a set of Pods.
```

Comment:
a ready-to-use Kubernetes configuration files => ready-to-use Kubernetes configuration files  (delete `a`)

Reply:
ok done